apiary.apib

```diff
@@ -407,8 +408,15 @@ A Post object has the following attributes:
 
     + Body
 
-            [
+            {
+                "first": "/surveys",
```

Comment:
That's how json-ld suggest to handle Collections. see https://www.w3.org/community/hydra/wiki/Pagination